docs/arch/monitor.md

```diff
@@ -0,0 +1,47 @@
+# Monitor
+
+## Contributors
+
+- Sarah THEOULLE
+- Pauline CONTAT
+- Thomas BROINE
+- Baptiste BRONSIN
+
+## Functionnalities
+
+- Listening events on a Git remote repository
+- Recognition of the event and adapt the action according to the type of the event
+- Calling the controller through an external API
+
```

Comment:
I think you're missing the user configuration part.

Reply:
I added the following line : "Retrieving CI configuration files from the remote Git repository"